packages/lua_parser/lua_parser.1.0.1/opam

```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A Lua 5.2 Parser"
+description: "This is a parser and pretty-printer for lua 5.2"
+maintainer: ["drjdnielsen@gmail.com"]
+authors: ["Jason D. Nielsen"]
+license: "MIT"
+homepage: "https://github.com/drjdn/ocaml_lua_parser"
+bug-reports: "https://github.com/drjdn/ocaml_lua_parser/issues"
+depends: [
+  "dune" {>= "2.5"}
+  "menhir" {>= "20200624"}
+  "sexp_pretty" {>= "v0.14.0"}
+  "sexplib" {>= "v0.14.0"}
+  "ppx_sexp_conv" {>= "v0.14.1"}
+  "ppx_deriving" {>= "4.5"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```
Tis is fixed if you use dune 2.7. Nothing is needed from your side, it is enough to fix it here